.travis.yml

```diff
@@ -1,12 +1,9 @@
 sudo: false
 language: erlang
 otp_release:
```

Comment:
@tsloughter @maximvl do you believe that OTP19 should be tested against as well?